src/AntiCSRF.php

```diff
@@ -94,17 +94,17 @@ public static function getTokenArray($lockto = null)
         }
 
         if (empty($lockto)) {
-            $lockto = isset($_SERVER['REQUEST_URI']) 
+            $lockto = isset($_SERVER['REQUEST_URI'])
                 ? $_SERVER['REQUEST_URI']
                 : '/';
         }
 
-        if (\preg_match('#/$#', $lockto)) {
```

Comment:
but that happens if you are in your namespace.

nobody should write "namespace ParagonIE\AntiCSRF" in their code.